subsys/bluetooth/services/Kconfig.dis

```diff
@@ -0,0 +1,109 @@
+# Kconfig - Bluetooth GATT Device Information service
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig BT_GATT_DIS
+	bool "Enable BLE GATT Device Information service"
+	default n
+
+if (BT_GATT_DIS)
+
+choice
+	prompt "Device Information service logging level"
+	default BT_GATT_DIS_LOG_LEVEL_INF
+
+ config BT_GATT_DIS_LOG_LEVEL_OFF
+	bool "Off"
+config BT_GATT_DIS_LOG_LEVEL_ERR
+	bool "Error"
+config BT_GATT_DIS_LOG_LEVEL_WRN
+	bool "Warning"
+config BT_GATT_DIS_LOG_LEVEL_INF
+	bool "Info"
+config BT_GATT_DIS_LOG_LEVEL_DBG
+	bool "Debug"
```

Comment:
This is inconsistent with how it is done usually right?

Reply:
True. AFAIK there's usually just a single integer option for the level with `range 0 4`